### PR TITLE
Generate Charges page

### LINKED ISSUE #50: Generate Charges Integration

### PR DESCRIPTION
**Pull request by anuj1511**

### CODE REVIEW

**Comment by anuj1511** [comment]
Generate charges is working now

File: pages/charges/generateCharges.js [hunk 1]
```diff
@@ -3,7 +3,122 @@ import DifferentCharges from "../../public/images/DifferentCharges.png";
...
+import getParticularSite from "../routeFunction/getParticularSite";
+import React, { useContext, useEffect, useState } from "react";
+import { Store } from "../../utility/Store";
+import Cookies from "js-cookie";
+import { useSnackbar } from "notistack";
+import { useRouter } from "next/router";
+
+import axios from "axios";
+
+export default function generateCharges() {
```
Comment (rishabhborn2win) [inline pages/charges/generateCharges.js]: Change the name of the function and start it with the capital letter

File: pages/routeFunction/getParticularSite.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// import React, { useContext } from "react";
```
Comment (rishabhborn2win) [inline pages/routeFunction/getParticularSite.js]: Why this file if you have to comment it all?

**Review by rishabhborn2win — CHANGES_REQUESTED** [review]

**Review by rishabhborn2win — APPROVED** [review]

**Comment by rishabhborn2win** [comment]
Screenshot 2022-01-29 at 10 39 36 PM
After Successful charges generation its redirected to a page where its showing site not found

**Comment by rishabhborn2win** [comment]
I think the Id that you have passed in the URL is not correct, its the database Id of the charge hence its showing site not found
Screenshot 2022-01-29 at 10 41 46 PM

**Pull request merged**